docs/v1/integrations/rest.mdx

```diff
@@ -0,0 +1,449 @@
+      "output": "Retrieved 5 users"
+    }
+  ]
+}
+```
+</CodeGroup>
+
+### Update Events
+
+Update existing events (e.g., adding completion information):
+
+<CodeGroup>
+```bash curl
+curl -X POST https://api.agentops.ai/update_events \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer your_jwt_token" \
+  -d '{
+    "events": [
+      {
+        "id": "event-id-123",
+        "completion": "Updated completion text",
+        "completion_tokens": 100
+      }
+    ]
+  }'
+```
+
+```bash Request
+POST https://api.agentops.ai/update_events
+Content-Type: application/json
+Authorization: Bearer your_jwt_token
+
+{
+  "events": [
+    {
+      "id": "event-id-123",
+      "completion": "Updated completion text",
+      "completion_tokens": 100
+    }
+  ]
+}
+```
+</CodeGroup>
+
+## Agent Management
+
+### Create Agent
+
+Register a new agent in a session:
+
+<CodeGroup>
+```bash curl
+curl -X POST https://api.agentops.ai/create_agent \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer your_jwt_token" \
+  -d '{
+    "id": "agent-123",
+    "name": "Research Assistant",
+    "role": "Performs web research",
+    "tools": ["web_search", "summarize"],
+    "system_prompt": "You are a research assistant..."
+  }'
+```
+
+```bash Request
+POST https://api.agentops.ai/create_agent
+Content-Type: application/json
+Authorization: Bearer your_jwt_token
+
+{
+  "id": "agent-123",
+  "name": "Research Assistant",
+  "role": "Performs web research",
+  "tools": ["web_search", "summarize"],
+  "system_prompt": "You are a research assistant..."
+}
+```
+</CodeGroup>
+
+## Thread Management
+
+### Create Thread
+
+Create a new conversation thread:
+
+<CodeGroup>
+```bash curl
+curl -X POST https://api.agentops.ai/create_thread \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer your_jwt_token" \
+  -d '{
+    "id": "thread-456",
+    "name": "Customer Inquiry",
+    "init_timestamp": "2024-03-14T12:00:00Z"
+  }'
+```
+
+```bash Request
+POST https://api.agentops.ai/create_thread
+Content-Type: application/json
+Authorization: Bearer your_jwt_token
+
+{
+  "id": "thread-456",
+  "name": "Customer Inquiry",
+  "init_timestamp": "2024-03-14T12:00:00Z"
+}
+```
+</CodeGroup>
+
+## Error Reporting
+
+### Report Developer Error
+
+Report implementation or runtime errors:
+
+<CodeGroup>
+```bash curl
+curl -X POST https://api.agentops.ai/developer_errors \
+  -H "Content-Type: application/json" \
+  -H "X-Agentops-Api-Key: your_api_key" \
+  -d '{
+    "error": "Invalid API response format",
+    "stack_trace": "...",
+    "context": {
+      "function": "process_response",
+      "line": 123
+    }
+  }'
+```
+
+```bash Request
+POST https://api.agentops.ai/developer_errors
+Content-Type: application/json
+X-Agentops-Api-Key: your_api_key
+
+{
+  "error": "Invalid API response format",
+  "stack_trace": "...",
+  "context": {
+    "function": "process_response",
+    "line": 123
+  }
+}
+```
+</CodeGroup>
```

Comment:
Don't need this either. This is us tracking stuff. We don't want people submitting BS reports